### DISCUSSION TITLE
C2 set to "Always Ready" still sometimes powers down (red light)

### DISCUSSION THREAD

**Discussion by djc5166** [JPersson77/LGTVCompanion#185] [Q&A]
Having some issues where even when my C2 is set to "Always Ready" and already turned off (white light) it sometimes decides to fully power off (red light). This happens seemingly at random, maybe once every couple of days, even when not doing anything that would cause LGTVC to send it any commands. ie. Just sitting at the windows desktop on my other display. It gets really annoying because it causes windows to reconfigure the displays as if you just disconnected one.

I don't think this really has anything to do with LGTVC, maybe it does? Has anyone else experienced something like this before?

**Comment by JPersson77** [comment]
Hi, the only time ive seen this happen is when a new firmware gets installed. I guess that is not what's happening on your system every few days, so i've no clue unfortunately

**Comment by djc5166** [comment]
[Log.txt](https://github.com/JPersson77/LGTVCompanion/files/13701988/Log.txt)
Here is some logging from the pc sitting mostly idle for a few days. Does this show you anything?

**Comment by JPersson77** [comment]
I've checked the log. It looks like the "other" display was active at the time, is that correct? I can see that some commands are sent to the LG, but it's also normal given that the TV seem to be disabled in the monitor topology. Maybe try to write down a timecode of when this happens and see if you can match that with an event in the log. I'm doubtful this issue has to do with the app at the moment, but let's try to find out.

**Comment by djc5166** [comment]
Yes there is another display connected, my main monitor.

I had posted and deleted a message.

As a test I had uninstalled LGTVC and kept the same configuration. It looked good at first but eventfully the TV still turned off (red) so I deleted my last comment.

So I don't think it is a LGTVC issue.
